helper_test.go

```diff
@@ -1233,18 +1233,17 @@ func waitForRunLock(t *testing.T, client *Client, workspaceID string) {
 	wg.Wait()
 }
 
-func retry(f retryableFn) (interface{}, error) {
+func retry(f retryableFn) (interface{}, error) { //nolint
 	tick := time.NewTicker(tickDuration * time.Second)
 
 	retries := 0
 	maxRetries := 5
 
 	for {
-		select {
-		case <-tick.C:
+		if tick.C != nil {
```

Comment:
To my eye, this doesn't look equivalent. `<-tick.C` should be getting a value from a channel, but comparing the channel to nil doesn't seem like it would have the same effect?